scriptworker/cot/generate.py

```diff
@@ -111,9 +118,14 @@ def generate_cot(context, path=None):
     )
     validate_json_schema(body, schema, name="chain of trust")
     body = format_json(body)
-    path = path or os.path.join(context.config['artifact_dir'], "public", "chainOfTrust.json.asc")
+    asc_path = path or os.path.join(context.config['artifact_dir'], "public", "chainOfTrust.json.asc")
+    unsigned_path = os.path.join(os.path.dirname(asc_path), 'chain-of-trust.json')
```

Comment:
+1. It looks like we might want to keep it since we use a non-tmpdir `artifact_dir` in some tests, but I could change `path` to `parent_path`, especially since we already have a good reason to bump the major version.